spec/controllers/bookmarks_controller_spec.rb

```diff
@@ -219,12 +219,46 @@ def it_redirects_to_user_login
     let(:bookmark) { create(:bookmark, bookmarkable_id: chaptered_work.id) }
 
     context "when logged in" do
-      it "returns a bookmark on a public multi-chapter work" do
+      before do
         fake_login_known_user(bookmark.pseud.user)
+      end
+
+      it "returns a bookmark on a public multi-chapter work" do
         get :show, params: { id: bookmark }
         expect(response).to have_http_status(:success)
         expect(assigns(:bookmark)).to eq(bookmark)
       end
+
+      it "finds a work from a bookmark" do
+        render_views
+        get :show, params: { id: bookmark }
+        expect(response).to have_http_status(:success)
+        expect(response.body).to include(chaptered_work.title)
```

Comment:
We don't generally render and check the views in specs. Could we check `expect(assigns(:work)).to eq(chaptered_work)` here instead? Or is this aiming for a line in the controller that requires the view?

Reply:
This part is solved now, found a way to check it